ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6332,6 +6332,17 @@ EFHLC:
   implements:
   - OPERATIONAL
 
+FHLC:
```

Comment:
Change to more general `HLC` if you can confirm the question on the fields section.